flask_app/blueprints/beams.py

```diff
@@ -20,35 +20,38 @@
 _ALLOWED_PARAMS = ['tag', 'pinned', 'uid', 'email', 'page']
 @beams.route('', methods=['GET'], strict_slashes=False)
 def get_all():
-    query = (
+    if any({param not in _ALLOWED_PARAMS for param in request.values}):    
+        abort(http.client.BAD_REQUEST)
+
+    beam_query = (
         db.session.query(Beam)
         .options(joinedload(Beam.pins), joinedload(Beam.type), joinedload(Beam.issues))
         .order_by(Beam.start.desc()))
-    if any({param not in _ALLOWED_PARAMS for param in request.values}):    
-        abort(http.client.BAD_REQUEST)
     for param in request.values:
+        if param == "page":
+            continue
         if param == "tag":
             tags = request.values['tag']
             for tag in tags.split(';'):
-                query = query.filter(Beam.tags.any(Tag.tag == tag))
+                beam_query = beam_query.filter(Beam.tags.any(Tag.tag == tag))
```

Comment:
This looks as before. Didn't we say that there was `order` instead of `filter` here?